pyserini/server/AnseriniApplication/__main__.py

```diff
@@ -0,0 +1,29 @@
+#
+# Pyserini: Reproducible IR research with sparse and dense representations
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+import time
+
+from pyserini.pyclass import autoclass
+
+if __name__ == '__main__':
+    args = sys.argv[1:]
+
+    JServer = autoclass('io.anserini.server.Application')
+    JServer.main(args)
+
+    while True:
+        time.sleep(60)
```

Comment:
Hrm... there's gotta be a better way to do this?

Reply:
I tried using signals, which should wait until it gracefully catches a terminating signal and stop the server. Hope that works better!